config.yaml

```diff
@@ -59,10 +68,8 @@ submodels:
       input:
         - /g/data/vk83/experiments/inputs/access-om2/ice/grids/global.01deg/2020.05.30/grid.nc
         - /g/data/vk83/experiments/inputs/access-om2/ice/grids/global.01deg/2020.05.30/kmt.nc
-        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions_biogeochemistry/global.01deg/2022.02.24/o2i.nc
-        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions_biogeochemistry/global.01deg/2022.02.24/i2o.nc
+        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions/global.1deg/2020.05.30/u_star.nc
```

Comment:
Yeah.. it was a typo. But I am not sure if it is actually required by the model. I adjusted these lines based on your comments to Dougie's PR. While removing `i2o.nc` and `o2i.nc`, I accidentally deleted the `u_star.nc`. So I mistakenlly copied `1deg` `u_star.nc` to my config.yaml. 

I will run another test without `u_star.nc` then compare, and let you know if it is required or not. Or is there a better way to see if `u_star.nc` is required?

Reply:
It seems that it is required, I have no idea why it worked with the 1 degree version. Or perhaps you didn't actually run it with that path? In which case all is fine.

This option is true:

https://github.com/COSIMA/01deg_jra55_ryf/blob/db8159373531de0f9237557308e179ddd4b69b1b/ice/input_ice.nml#L18

Which implies it is read in:

https://github.com/COSIMA/cice5/blob/edcfa6f9c76ed05b63196ce4b5355fa5a8f4fe3a/drivers/auscom/CICE_InitMod.F90#L206-L208

and would fail if it doesn't exist

https://github.com/COSIMA/cice5/blob/edcfa6f9c76ed05b63196ce4b5355fa5a8f4fe3a/drivers/auscom/cpl_forcing_handler.F90#L278-L305

Reply:
I leave `u_star.nc` path as it is now. 

`- /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions/global.01deg/2020.05.30/u_star.nc`